README.md

```diff
@@ -1,40 +1,148 @@
+<!-- nextcloud-talk-jitzsi-bot README.md -->
+<!-- version: 0.1.1 -->
+
 # Nextcloud Talk Jitsi Bot
 
-A chat bot for Nextcloud Talk that creates [Jitsi](https://jitsi.org) video chat meetings. It also includes a framework for building Nextcloud Talk chatbots.
+A chat bot for Nextcloud Talk that creates [Jitsi](https://jitsi.org) video chat meetings.
+It also includes a framework for building Nextcloud Talk chatbots.
 
 ## Overview
 
-Nextcloud Talk Jitsi Bot creates ad-hoc Jitsi meetings from Nextcloud Talk. Just add the bot to the conversation, type `#videocall` or `#videochat` and get a link to a Jitsi meeting!
+Nextcloud Talk Jitsi Bot creates ad-hoc Jitsi meetings from Nextcloud Talk sessions.
+
+The talk sessions needs to be aware of the bot. Just create a new `Jitsi-Bot user` inside your trusted `nextcloud` domain.
+Its name needs to reflact the `BOT_NEXTCLOUD_USERNAME` environment variable. The bot binary
+will resolve it and interact via the `Nextcloud API` to the `Talk` sessions.
+
+When you start a talk session, please add the `Jitsi-Bot user` as a participant of your session.
+Once done, you can type `#videocall` or `#videochat`. A newly created link to a Jitsi meeting will be advertised!
 
 ## Installation
 
-### Go Package
+Precompiled binaries are provided from the following locations.
 
-A Go package [is available](https://pkg.go.dev/mod/github.com/pojntfx/nextcloud-talk-jitsi-bot).
+### Go module
+
+A Go package is available from githubs module repro [pkg.go.dev](https://pkg.go.dev/mod/github.com/pojntfx/nextcloud-talk-jitsi-bot).
 
 ### Docker Image
 
-A Docker image is available on [Docker Hub](https://hub.docker.com/r/pojntfx/nextcloud-talk-jitsi-bot).
+A Docker image is available at [Docker Hub](https://hub.docker.com/r/pojntfx/nextcloud-talk-jitsi-bot).
+
+Pull the image
+
+	docker pull hub.docker.com/pojntfx/nextcloud-talk-jitsi-bot
+
+### OCI Image
+
+An Open Containers Initiative (OCI) image is available at
+[Quay.IO](https://quay.io/rzerres/nextcloud-talk-jitsi-bot).
+
+Pull the image
+
+	podman pull quay.io/rzerres/nextcloud-talk-jitsi-bot:latest
+
+<!--
+#### Howto uploading to Quay.io
```

Comment:
Maybe move this to something like `CONTRIBUTING.md`?

Reply:
> Maybe move this to something like `CONTRIBUTING.md`?

I you like to split the README.md thing, it would be better suited in INSTALL.md. Since all this goes to instructions depending on the public repro porvider. I you are more keen with a single repo, OCI generated images should be compatible whith docker usage. I just didn't test and didn't want to interfere with your working docker stuff ....